packages/style-loader/index.js

```diff
@@ -44,9 +44,9 @@ module.exports = (neutrino, opts = {}) => {
         test: options.modulesTest,
         exclude: options.modulesExclude,
         ruleId: `${options.ruleId}${options.modulesSuffix}`,
-        styleUseId: `${options.styleUseId}${options.modulesSuffix}`,
-        cssUseId: `${options.cssUseId}${options.modulesSuffix}`,
-        extractId: `${options.extractId}${options.modulesSuffix}`,
+        styleUseId: `${options.styleUseId}`,
```

Comment:
These don't need to be wrapped in template strings I think.

Reply:
In fact, they don't need to be there at all, now that the ids are the same…